examples/flows/standard/conditional-flow-for-if-else/llm_result.py

```diff
@@ -0,0 +1,9 @@
+from promptflow import tool
+
+
+@tool
+def llm_result(request: str) -> str:
```

Comment:
Add comment: use an LLM node to replace this tool

Reply:
Updated.